tests/Oryx.BuildImage.Tests/Node/NodeDynamicInstallationTest.cs

```diff
@@ -442,6 +443,7 @@ public void GeneratesScript_AndBuildNodeAppsWithDynamicInstallationOnBullseyeIma
         [Theory, Trait("category", "githubactions")]
         [Trait("build-image", "github-actions-debian-bookworm")]
         [InlineData("20.9.0", ImageTestHelperConstants.GitHubActionsBookworm)]
+        [InlineData("22.9.0", ImageTestHelperConstants.GitHubActionsBookworm)]
```

Comment:
It might be a good idea to make use of constants here so we don't have to manually update test cases everytime

Reply:
yes, we can take that for next pr